src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -4932,7 +4838,7 @@ ITermDispatch::StringHandler AdaptDispatch::_RestoreTabStops()
 // - params - The volume, duration, and note values to play.
 // Return value:
 // - True if handled successfully. False otherwise.
-bool AdaptDispatch::PlaySounds(const VTParameters parameters)
+void AdaptDispatch::PlaySounds(const VTParameters parameters)
 {
```

Comment:
I can't highlight the actual line, but there's another unnecessary `return` below.